### ISSUE TITLE
Connio3.0 - Cannot read device state 

### ISSUE THREAD

**Issue by volkansk** [connio/octopus-issues#205]
Hi,
We want to read device state details of the two devices under the -ttbulutolcum- subaccount in Connio 3.0. But Timeout Error is returning. Could you check it please?

**Devices:** _dev_386732294207241209 / _dev_409042655707934338

**ttbulutolcum user credentials :** 
```
url: https://api3.inv.connio.net/v3
user: _key_378019019262550444
pass: 25f18e2e29654144b0210c059ced110c
```

**Request:**
`https://api3.inv.connio.net/v3/accounts/_acc_378018993278836130/data/devices/_dev_386732294207241209/state`

**Response:**
```
[
    {
        "code": "TimeoutError$",
        "message": "System could not reply to your request in expected time",
        "friendlyMessage": "Please try again later.",
        "moreInfoUrl": "https://docs.connio.com/errors/timeouterror$"
    }
]
```

**Comment by jcroig** [comment]
Have you been writing big property values to it?

**Comment by volkansk** [comment]
The enviroment is prod in Connio 3.0.

**Comment by jcroig** [comment]
Have you been writing big property values to this device?

**Comment by skaranuh** [comment]
No. This device firmware has no operation regarding big property values.

**Comment by bmrb** [comment]
It looks like the data cannot be read from the datastore even using datastore native tools (i.e. `cqlsh`). Just recreated devices.
